tools/ci_build/build.py

```diff
@@ -165,7 +165,7 @@ def parse_arguments():
     parser.add_argument("--use_dml", action='store_true', help="Build with DirectML.")
     parser.add_argument("--use_winml", action='store_true', help="Build with WinML.")
     parser.add_argument("--use_telemetry", action='store_true', help="Only official builds can set this flag to enable telemetry.")
-    parser.add_argument("--enable_wcos", action='store_true', help="Build for Windows Core OS.")
+    parser.add_argument("--win32", action='store_true', help="Build for Win32 instead of Windows Core OS APIs")
```

Comment:
we had agreed a little earlier on that the default behavior for the build should be win32, as there are other pipeline build (like for edge) that require a win32 and are not explicitly configured for win32.